src/FluidClassBuilder-Tests/FluidClassBuilderTest.class.st

```diff
@@ -87,7 +87,24 @@ FluidClassBuilderTest >> testBuilderSharedVariables [
 FluidClassBuilderTest >> testBuilderTraits [
 	
 	builder uses: TViewModelMock.
-	self assert: builder traitsToBuild equals: TViewModelMock 
+	self assert: builder traitsToBuild equals: TViewModelMock.
+	self shouldnt: [ builder fillShiftClassBuilder ] raise: MessageNotUnderstood 
```

Comment:
Why not allow test to simply fail? 
I am sure that good books describe such assertions as an anti pattern. 
 
With this style you will not get a normal debugger in case of DNU. And error log will miss a lot of contextual information (depends how shouldnot:raise: is implemented). Therefore it will complicate resolving the bug